lib/search.js

```diff
@@ -68,8 +68,14 @@ function onFlowActionSearchLive (callback, args) {
   exports.searchQuery(args.query, function (err, result) {
     if (err) return Homey.error(err)
 
+    var itemNumber = 0
+    if (args.mode === 'random') {
+      // Select a random video
+      var maxRandom = result.items.length
+      itemNumber = Math.floor(Math.random() * (maxRandom + 1))
```

Comment:
Yep you're right. It has to be exclusive because of the 0-index, been working with too many 1-index based languages lately :). I'll change it tonight.